GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -859,6 +832,10 @@ int IndexOf(string s)
 
                     return -1;
                 }
+
+                // Note: This assumes that "master" is the important branch, this is not configurable.
+                bool IsLocalMaster(string branch) => branch == "master";
```

Comment:
New suggestion: Can this be set to all branches starting with "master"? 
That would improve my use case and I cannot see that this will be too cumbersome for other users.
(IsRemoteMaster will require a regexp)

Reply:
Done. Would have been no big deal if identifiers like `aIsRemoteMaster` would not have been denied as Hungarian Notation. So dont blame me for `isRemoteMaster_a`!

![grafik](https://user-images.githubusercontent.com/36601201/67906759-e8fab280-fb75-11e9-9894-f06eab17a9cb.png)

Reply:
> That would improve my use case and I cannot see that this will be too cumbersome for other users.

omg, Gerhard-specific feature 😁